mock/cpp_mock_scenarios/src/crosswalk/stop_at_crosswalk.cpp

```diff
@@ -30,9 +29,7 @@ class StopAtCrosswalkScenario : public cpp_mock_scenarios::CppScenarioNode
 {
 public:
   explicit StopAtCrosswalkScenario(const rclcpp::NodeOptions & option)
-  : cpp_mock_scenarios::CppScenarioNode(
-      "stop_at_crosswalk", ament_index_cpp::get_package_share_directory("kashiwanoha_map") + "/map",
```

Comment:
.osm file should be exactly fixed in scenario.
So, please specify it in constructor.